source/appModules/foobar2000.py

```diff
@@ -54,9 +55,14 @@ def getElapsedAndTotal(self):
 		statusBarContents = self._statusBar.firstChild.name
 		try:
 			playingTimes = statusBarContents.split("|")[4].split("/")
-			return statusBarTimes(playingTimes[0], playingTimes[1])
 		except IndexError:
 			return empty
+		elapsed = playingTimes[0]
+		if len(playingTimes) > 1:
+			remaining = playingTimes[1]

```

Comment:
This function and the named tuple `statusBarTimes` refer to elapsed and total. This line refers to remaining, this could easily confuse a future developer.

Is this value supposed to be total time?